### PR TITLE
added import for 360

### PR DESCRIPTION
**Pull request by DavidGeismarLtd**

### CODE REVIEW

File: app/views/admin/programs/_collection.html.erb [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<%#
```
Comment (ManyBaby) [inline app/views/admin/programs/_collection.html.erb:1]: do we really need this view, it seems we are not doing anything with it ?

**Review by ManyBaby — COMMENTED** [review]
Efficient way of getting fresh content from 360 learning on demand.
I'm just wondering how the diff works. If your resync do you loose everything JP Djian has been updating for days ?

File: app/views/admin/programs/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  <% if show_search_bar %>
+    <%= render(
+      "search",
+      search_term: search_term,
+      resource_name: display_resource_name(page.resource_name)
+    ) %>
+  <% end %>
+
+  <div>
+    <%= link_to(
```
[the remark this answers is not in the document]
Comment (DavidGeismarLtd) [reply app/views/admin/programs/index.html.erb:44]: yep

File: app/views/admin/programs/_collection.html.erb (hunk 1 above)
Comment (DavidGeismarLtd) [reply app/views/admin/programs/_collection.html.erb:1]: I think you are right we dont need it, ill remove it

**Pull request merged**